content/en/docs/concepts/services-networking/service.md

```diff
@@ -1178,13 +1186,17 @@ mechanism Kubernetes provides to expose a Service with a virtual IP address.
 
 ## {{% heading "whatsnext" %}}
 
-Learn more about the following:
-* Follow the [Connecting Applications with Services](/docs/tutorials/services/connect-applications-service/) tutorial
-* [Ingress](/docs/concepts/services-networking/ingress/) exposes HTTP and HTTPS routes from outside the cluster to services within the cluster.
-* [EndpointSlices](/docs/concepts/services-networking/endpoint-slices/)
+Learn more about Services and how they fit into Kubernetes:
+* Follow the [Connecting Applications with Services](/docs/tutorials/services/connect-applications-service/) tutorial.
+* Read about [Ingress](/docs/concepts/services-networking/ingress/), which
+  exposes HTTP and HTTPS routes from outside the cluster to Services within
+  your cluster.
+* Read about [Gateway](https://gateway-api.sigs.k8s.io/), an extension to
+  Kubernetes that provides more flexibility than Ingress.
 
-For more context:
+For more context, read:
```

Comment:
```suggestion
For more context, read the following:
```

Reply:
OK, but why?

Reply:
My reasoning was not to have the introductory fragment be completed by the items in the list per

https://developers.google.com/style/lists#introductory-sentences-for-lists

But Microsoft's style guide suggests that it might make localization more difficult as well

https://learn.microsoft.com/en-us/style-guide/scannable-content/lists#introductory-text

Reply:
Yes, the suggestion makes sense. You could go further and state that more information is found in the API reference for each resource (or something like that 😃 ).